src/SavedSearch.php

```diff
@@ -1323,11 +1323,7 @@ public function execute($force = false, bool $enable_partial_warnings = true)
 
             $params = null;
             if (class_exists($this->getField('itemtype'))) {
-                $params = $this->prepareQueryToUse(
-                    $this->getField('type'),
-                    $query_tab,
-                    $enable_partial_warnings
-                );
+                $params = $search::manageParams($this->getField('itemtype'), $_GET);
```

Comment:
I do not know if this fix is correct, but it should probably set the `$usesession` parameter to false.

```suggestion
                $params = $search::manageParams($this->getField('itemtype'), $_GET, false);
```